lib/widget/camera.dart

```diff
@@ -89,17 +95,24 @@ class CameraScreenState extends WidgetState<CameraScreen> with WidgetsBindingObs
     errorString = 'You just pick at least ${widget._controller.maxCount} image';
     initCamera().then((_) {
       ///initialize camera and choose the back camera as the initial camera in use.
-      if (cameras.length == 2) {
-        if (widget._controller.initialCamera == 'back') {
-          setCamera(1);
+      if (cameras.length >= 2) {
+        if (widget._controller.initialCamera == InitialCamera.back) {
+          final back = cameras.firstWhere(
+              (camera) => camera.lensDirection == CameraLensDirection.back);
+          setCamera(cameraDescription: back);
           isFrontCamera = false;
-        } else {
-          setCamera(0);
+          setState(() {});
```

Comment:
these lines are repeated three times in this block. Can you simplify it.

Reply:
When initial camera gets back then we can find description of back camera through list and pass to set camera then that camera runs. Same goes for front camera and in case of no back camera and no front camera then default. If user doesn’t add initial camera option then it will go back by default. But this process will happen when camera length is 2 or greater than 2. Otherwise it will open default camera which will run isdefine that can be front or back